core/modules/filters.js

```diff
@@ -62,43 +62,61 @@ function parseFilterOperation(operators,filterString,p) {
 		else if(operator.operator === "") {
 			operator.operator = "title";
 		}
+		operator.operands = [];
+		function parseOperand(bracketType) {
+			var operand = {};
+			switch (bracketType) {
+				case "{": // Curly brackets
+					operand.indirect = true;
+					nextBracketPos = filterString.indexOf("}",p);
+					break;
+				case "[": // Square brackets
+					nextBracketPos = filterString.indexOf("]",p);
+					break;
+				case "<": // Angle brackets
+					operand.variable = true;
+					nextBracketPos = filterString.indexOf(">",p);
+					break;
+				case "/": // regexp brackets
+					var rex = /^((?:[^\\\/]*|\\.)*)\/(?:\(([mygi]+)\))?/g,
+						rexMatch = rex.exec(filterString.substring(p));
+					if(rexMatch) {
+						operator.regexp = new RegExp(rexMatch[1], rexMatch[2]);
+	// DEPRECATION WARNING
+	console.log("WARNING: Filter",operator.operator,"has a deprecated regexp operand",operator.regexp);
+						nextBracketPos = p + rex.lastIndex - 1;
+					}
+					else {
+						throw "Unterminated regular expression in filter expression";
+					}
+					break;
+			}
 
-		p = nextBracketPos + 1;
-		switch (bracket) {
-			case "{": // Curly brackets
-				operator.indirect = true;
-				nextBracketPos = filterString.indexOf("}",p);
-				break;
-			case "[": // Square brackets
-				nextBracketPos = filterString.indexOf("]",p);
-				break;
-			case "<": // Angle brackets
-				operator.variable = true;
-				nextBracketPos = filterString.indexOf(">",p);
-				break;
-			case "/": // regexp brackets
-				var rex = /^((?:[^\\\/]*|\\.)*)\/(?:\(([mygi]+)\))?/g,
-					rexMatch = rex.exec(filterString.substring(p));
-				if(rexMatch) {
-					operator.regexp = new RegExp(rexMatch[1], rexMatch[2]);
-// DEPRECATION WARNING
-console.log("WARNING: Filter",operator.operator,"has a deprecated regexp operand",operator.regexp);
-					nextBracketPos = p + rex.lastIndex - 1;
-				}
-				else {
-					throw "Unterminated regular expression in filter expression";
-				}
-				break;
-		}
-
-		if(nextBracketPos === -1) {
-			throw "Missing closing bracket in filter expression";
-		}
-		if(!operator.regexp) {
-			operator.operand = filterString.substring(p,nextBracketPos);
+			if(nextBracketPos === -1) {
+				throw "Missing closing bracket in filter expression";
+			}
+			if(!operator.regexp) {
+				operand.text = filterString.substring(p,nextBracketPos);
+				operator.operands.push(operand);
+			}
+			p = nextBracketPos + 1;
 		}
+		
 		p = nextBracketPos + 1;
-
+		parseOperand(bracket);
+		
+		// Check for multiple operands
+		while(filterString.charAt(p) === ",") {
```

Comment:
Operands can be {!!title}. right? ... Since titles can have commas, is there a problem?

Reply:
@pmario no this isn't an issue as the parser reads the complete operand before checking for the comma. I did test this but an explicit test is a good idea and I will add one.